packages/babel-preset-gatsby-package/package.json

```diff
@@ -8,15 +8,14 @@
     "directory": "packages/babel-preset-gatsby-package"
   },
   "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/babel-preset-gatsby-package#readme",
-  "peerDependencies": {
-    "@babel/core": "^7.0.0"
-  },
   "dependencies": {
+    "@babel/core": "^7.6.6",
```

Comment:
This should stay a peerdependency. Babel presets shouldn't have a dependency to babel itself.